docs/setup-development.md

```diff
@@ -0,0 +1,235 @@
+
+replace `sessionTokenHere==` with the string of characters between the apostrophes (' ') above so you end up with something that looks like this in that field:
+
+> Token SWpFaS5EaEoxRlEubHRVC1DSTVJZ2hfalMc0xlalu3KRk5BUGk0
+
+Your user must have logged in to the local testing instance once of course and have the needed permissions for the API call.
+
+You can get your OSM user id number either by finding it in your local testing/dev database `select * from users` or from OSM by viewing the edit history of your user, selecting a changeset from the list, and then at the bottom link `Changeset XML` and it will be in the `uid` field of the XML returned.
+
+To get your token on the production or staging Tasking Manager instances, after sign in in the browser, inspect a network request and search for the `Authorization` field in the request headers section.
+
+### DevOps
+If you encounter any issues while setting up a dev environment, please visit our [FAQ ❓ page](./setup-development-faqs.md) to find possible solutions.
+
+# Docker
+
+## Creating a local PostGIS database with Docker
+
+If you're not able to connect to an existing tasking-manager DB, we have a [Dockerfile]() that will allow you to run PostGIS locally as follows.
+
+### Dependencies
+
+Following must be available locally:
+
+* [Docker CE](https://www.docker.com/community-edition#/download)
+
+### Build & Run the PostGIS dockerfile
+
+1. From the root of the project:
+
+`
+docker build -t tasking-manager-db ./devops/docker/postgis
+`
+
+2. The image should be downloaded and build locally.  Once complete you should see it listed, with
+
+`
+docker images
+`
+
+3. You can now run the image (this will run PostGIS in a docker container, with port 5432 mapped to localhost):
+
+`
+docker run -d -p 5432:5432 tasking-manager-db
+`
+
+4.  Confirm the image is running successfully:
+
+`
+docker ps
+`
+
+5. Finally you can set your env variable to point at your containerised DB:
+
+`
+export TM_DB=postgresql://hottm:hottm@localhost/tasking-manager
+`
+
+6.  Refer to the rest of the instructions in the README to setup the DB and run the app
+
+# Non-Docker
+
+## Creating the PostGIS database
+
+It may be the case you would like to set up the database without using Docker for one reason or another. This provides you with a set of commands to create the database and export the database address to allow you to dive into server development.
+
+### Dependencies
+
+First, ensure that Postgresql and PostGIS are installed and running on your computer.
+
+### Create the database user and database
+
+Assuming you have sudo access and the unix Postgresql owner is `postgres`:
+
+```
+$ sudo -u postgres psql
+$ CREATE USER "hottm" PASSWORD 'hottm';
+$ CREATE DATABASE "tasking-manager" OWNER "hottm";
+$ \c "tasking-manager";
+$ CREATE EXTENSION postgis;
+```
+
+Finally, add the environmental variable to access the database:
+
+`
+export TM_DB=postgresql://hottm:hottm@localhost/tasking-manager
```

Comment:
I think that TM_DB variable is outdated. On the top of the page it recommends another set of env vars.

Reply:
`TM_DB` still works, but the other ones should be mentioned here. However this PR is more about moving the pages in. I agree with you that we have to go through the whole documentation and update it. This PR can only be a first step.